packages/react-devtools-shared/src/backend/utils.js

```diff
@@ -188,7 +188,7 @@ export function formatWithStyles(
   }
 
   // Matches any of %(o|O|d|i|s|f), but not %%(o|O|d|i|s|f)
-  const REGEXP = /([^%]|^)(%([oOdisf]))/g;
+  const REGEXP = /(?<!%)((%%)*)(%([oOdisf]))/g;
```

Comment:
:( lookbehind is not supported in Safari https://caniuse.com/js-regexp-lookbehind